pyproject.toml

```diff
@@ -128,9 +128,10 @@ disable = [
   "isinstance-second-argument-not-valid-type", # conflict with ruff
   "fixme", # we're still developing
   "too-few-public-methods", # unavoidable
-  "too-many-positional-arguments", # unavoidable"
 ]
 
+max-positional-arguments = 10
```

Comment:
It should go under design category:

https://pylint.pycqa.org/en/latest/user_guide/configuration/all-options.html#max-positional-arguments

Reply:
Yes, I know, but that doesn't work. So I think this is a bug in pylint then ?

Reply:
Did you try it?

Reply:
This does NOT work:
![Scherm­afbeelding 2024-09-23 om 13 07 48](https://github.com/user-attachments/assets/d1997b98-fc83-40ec-82d2-b73aa80cfd29)

While this DOES work:
![Scherm­afbeelding 2024-09-23 om 13 08 07](https://github.com/user-attachments/assets/50034b9c-9d76-4c83-b6a8-9fba7f6a48dc)

Reply:
It's not called "too-many...".

Reply:
> It's not called "too-many...".

sigh.... I'm going to grab coffee now.

Reply:
works as a charm @MartinHjelmare !